storybook/src/utils/VisuallyHidden/VisuallyHidden.stories.tsx

```diff
@@ -3,25 +3,16 @@
  * Copyright Gemeente Amsterdam
  */
 
-import { Paragraph, VisuallyHidden } from '@amsterdam/design-system-react/src'
 import { Meta, StoryObj } from '@storybook/react'
+import type { HTMLAttributes } from 'react'
+
+type VisuallyHiddenProps = HTMLAttributes<HTMLSpanElement>
+
+const VisuallyHidden = ({ children }: VisuallyHiddenProps) => <span className="ams-visually-hidden">{children}</span>
 
 const meta = {
-  title: 'Components/Containers/Visually Hidden',
+  title: 'Utilities/CSS/Visually Hidden',
```

Comment:
Do we expect other utilities than utility classes? If not, I'd just call this section 'Utility classes'

Reply:
Yes, I expect we can offer JS formatting functions for the way Amsterdam wants to display numbers, dates etc.

And removing this level will promote the utility classes to a very visible spot that they don’t really deserve.

Reply:
I'd put this under docs too, it is confusing because it's not a component but does have an html code example.